specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/examples/CosmosDBRestorableSqlContainerList.json

```diff
@@ -0,0 +1,102 @@
+{
+  "parameters": {
+    "api-version": "2020-06-01-preview",
+    "subscriptionId": "subid",
+    "location": "WestUS",
+    "instanceId": "98a570f2-63db-4117-91f0-366327b7b353",
+    "restorableSqlDatabaseRid": "3fu-hg=="
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/subid/providers/Microsoft.DocumentDb/locations/westus/restorableDatabaseAccounts/98a570f2-63db-4117-91f0-366327b7b353/restorableSqlContainers/79609a98-3394-41f8-911f-cfab0c075c86",
+            "type": "Microsoft.DocumentDB/locations/restorableDatabaseAccounts/restorableSqlContainers",
+            "name": "79609a98-3394-41f8-911f-cfab0c075c86",
+            "properties": {
+              "resource": {
+                "_rid": "zAyAPQAAAA==",
+                "eventTimestamp": "2020-10-13T04:56:42Z",
+                "ownerId": "Container1",
```

Comment:
>ownerId [](start = 17, length = 7)

Can we rename owner to something else, it is not clearly indicative of its nature. How does something like, originalContainerId, sound.

Reply:
For now, I think it is okay to keep it as is since we have the description and also the value ("Container1") should be indicative that's it's a collection/container name.
We can improve on the name in the next version, since this name is currently getting deployed.